pym/_emerge/depgraph.py

```diff
@@ -1555,6 +1555,8 @@ def __str__(self):
 						unexplored.add(child)
 					else:
 						unexplored_tuples.add(child)
+                                if not unexplored:
+                                    break
```

Comment:
install app-vim/editorconfig-vim then the .editorconfig will configure you for tab indents

Reply:
Sometimes it's necessary to exit a while loop with a break, but in this case I think it makes the code less readable. The cost of checking the size of `unexplored` should be negligible.